knowledge-base/drawer-click-in-dropdown-closes-drawer.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Click in dropdown closes the Drawer
+description: Click in dropdown closes the Drawer
+type: troubleshooting
+page_title: Click in dropdown closes the Drawer
+slug: drawer-kb-click-in-dropdown-closes-drawer
+position: 
+tags: drawer, dropdown, click, close
+ticketid: 1575100
+res_type: kb
+---
+
+## Environment
+
+<table>
+    <tr>
+        <td>Product</td>
+        <td>Drawer for Blazor</td>
+    </tr>
+    <tr>
+        <td>Product</td>
+        <td>AutoComplete for Blazor, ComboBox for Blazor, DropDownList for Blazor, MultiSelect for Blazor</td>
+    </tr>
+    <tr>
+        <td>Product</td>
+        <td>DatePicker for Blazor, DateRangePicker for Blazor, DateTimePicker for Blazor, DateTimePicker for Blazor</td>
+    </tr>
+</table>
```

Comment:
This structure with repeating Product looks a bit confusing. Can we present the information in a bit different way?